stdlib/ballerina-builtin/src/main/ballerina/runtime/natives.bal

```diff
@@ -41,3 +41,11 @@ public native function getCurrentDirectory () returns (string);
 @Description {value:"Returns the charset encoding used in the runtime."}
 @Return { value:"Encoding if it is available, an empty string otherwise"}
 public native function getFileEncoding () returns (string);
+
+@Description {value:"Executes a ballerina command and returns the input stream connected to the normal and error
+output of the subprocess"}
+@Param {value:"command: The command to be executed. It should be one of run, docker, build, install, uninstall, pull,
+push, init, search, doc, grpc, swagger, test, version and encrypt"}
+@Param {value:"data: Name of the package"}
+@Return {value:"Data piped from the standard output and error output of the process"}
+public native function execBallerina (string command, string data) returns (string);
```

Comment:
'command' and 'data' must be annotated "@ sensitive" due to security sensitive nature of both elements. The return value should be "@ tainted".